kotsadm/pkg/apiserver/bootstrap.go

```diff
@@ -10,5 +12,48 @@ func bootstrap() error {
 		return errors.Wrap(err, "failed to init store")
 	}
 
+	if err := bootstrapClusterToken(); err != nil {
+		return errors.Wrap(err, "failed to bootstrap cluster token")
+	}
+
+	if err := bootstrapSharedPassword(); err != nil {
+		return errors.Wrap(err, "failed to bootstrap shared password")
+	}
+
+	return nil
+}
+
+func bootstrapClusterToken() error {
+	if os.Getenv("AUTO_CREATE_CLUSTER_TOKEN") == "" {
+		return errors.New("AUTO_CREATE_CLUSTER_TOKEN is not set")
+	}
+
+	_, err := store.GetStore().LookupClusterID("ship", "this-cluster", os.Getenv("AUTO_CREATE_CLUSTER_TOKEN"))
```

Comment:
How would you feel about just calling `ListClusters` and having less code here? 
This could iterate through them, and use the existing store function.

Reply:
@marccampbell `ListClusters` returns cluster IDs, so we'd have to do separate lookup for each

Reply:
Well ok then.  Fair enough.  `ListClusters` should be returning full cluster objects (`ListInstalledApps` does). But thats out of scope